src/app/components/Spinner/index.tsx

```diff
@@ -1,28 +1,37 @@
 import React from 'react';
-import { Spinner as BPSpinner } from '@blueprintjs/core/lib/esm/components/spinner/spinner';
 import cn from 'classnames';
+import { SpinnerSizeType } from './types';
+import styles from './index.module.scss';
 
-interface Props {
+interface ISpinnerProps {
   className?: string;
-  innerClassName?: string;
+  size?: SpinnerSizeType;
 }
 
-export const Spinner: React.FC<Props> = ({ className, innerClassName }) => {
+export const Spinner: React.FC<ISpinnerProps> = ({ className, size }) => {
   return (
-    <div
-      className={cn(
-        'tw-flex tw-flex-row tw-items-center tw-justify-center',
-        className,
-      )}
-    >
-      <span
-        className={cn(
-          'tw-w-min tw-btn-loader__spinner tw-flex tw-flex-row tw-items-center active',
-          innerClassName,
-        )}
+    <div className={cn(styles.spinner, className)}>
+      <svg
+        className="tw-animate-spin-fast"
+        width={size || SpinnerSizeType.STANDARD}
+        height={size || SpinnerSizeType.STANDARD}
```

Comment:
```suggestion
        width={size}
        height={size}
```

Reply:
set to the standard size in props by default